tests/schema/test_mutation.py

```diff
@@ -214,12 +213,10 @@ class Input:
     class Mutation:
         @strawberry.mutation
         def say(self, input: Input) -> str:
-            data = dataclasses.asdict(input)
-
-            if is_unset(data["name"]):
+            if is_unset(input.name):
```

Comment:
Why this change?

Reply:
when you run `dataclass.asdict(input)`, for some strange reason (idk if this is intented) it initializes a new class for each type, that is, as `name` field is of the type `UNSET`, it initializes a new instance of the class `_Unset` and therefore the function `is_unset` is false. Basically, is the for the test case to pass as I did this change:
https://github.com/strawberry-graphql/strawberry/blob/429faeec5db2ce061fba208c150e36314c58294c/strawberry/types/arguments.py#L31-L33

and previously it was https://github.com/strawberry-graphql/strawberry/blob/ba6f79a6cb17efb4918551225bd6c39a53e117ef/strawberry/arguments.py#L29-L30

I did that change to avoid multiple instances of the constant `UNSET` which is actually an instance of the class `_Unset`